src/main/java/uk/co/evoco/webdriver/utils/RadioButtonUtils.java

```diff
@@ -25,14 +27,18 @@ public static void selectByLabel(List<WebElement> webElements, String visibleLab
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes